### ISSUE TITLE
ENH: Add mf_bads option

### ISSUE THREAD

**Issue by larsoner** [mne-tools/mne-bids-pipeline#760]
One solution that should work for this would be something like:
```
mf_bads: Literal["run", "union"] = "run"
```
where changing to `"union"` it will use the union of all bads across all data that will be maxwell filtered (runs + noise + rest). This actually in some sense is a safer option because the autobad only sometimes finds bad channels that it should find, so by combining across runs with a union is probably safest / most conservative.

_Originally posted by @larsoner in https://github.com/mne-tools/mne-bids-pipeline/issues/759#issuecomment-1623806155_

**Comment by hoechenberger** [comment]
Makes sense to me, but I wouldn't label it a bug fix but rather a behavior change
